api/test/shell/cli_test.sh

```diff
@@ -46,7 +46,7 @@ fi
 
 #change level and path
 
-sed -i 's/file_path: logs\/error.log/file_path: .\/error.log/' conf/conf.yaml
```

Comment:
one PR for one thing, please revert this commit

Reply:
cc @nic-chen

Reply:
reverted.